controller/array_action/errors.py

```diff
@@ -143,20 +143,20 @@ def __init__(self, lun, host):
 
 class MappingError(BaseArrayActionException):
 
-    def __init__(self, vol, host, err):
-        self.message = messages.MappingError_message.format(vol, host, err)
+    def __init__(self, volume, host, err):
+        self.message = messages.MappingError_message.format(volume, host, err)
 
 
 class VolumeAlreadyUnmappedError(BaseArrayActionException):
 
-    def __init__(self, vol):
-        self.message = messages.VolumeAlreadyUnmapped_message.format(vol)
+    def __init__(self, volume):
+        self.message = messages.VolumeAlreadyUnmapped_message.format(volume)
 
 
 class UnMappingError(BaseArrayActionException):
 
-    def __init__(self, vol, host, err):
-        self.message = messages.UnMappingError_message.format(vol, host, err)
+    def __init__(self, volume, host, err):
+        self.message = messages.UnMappingError_message.format(volume, host, err)
 
```

Comment:
it's not the actual volume that sent just name or id.. do you think it should be reflected there?

Reply:
from the error class perspective, it might also be ok to pass a printable volume object, but I'm not sure

Reply:
true, but there is the `id_or_name` parameter, so what the diff? could we unite it?

Reply:
I don't see `VolumeAlreadyUnmappedError` being called with id (or at all by ds8k), are you sure it's not too soon?

Reply:
done (+ fixed ds8k)

Reply:
tnx